packages/api/lib/testUtils.js

```diff
@@ -395,6 +396,27 @@ async function getSqsQueueMessageCounts(queueUrl) {
   };
 }
 
+const createRuleTestRecords = async (context, ruleParams) => {
+  const {
+    testKnex,
+    ruleModel,
+    rulePgModel,
+  } = context;
+  const originalRule = fakeRuleFactoryV2(ruleParams);
+
```

Comment:
Do we need to create the rule trigger before creating the rule?

Reply:
Sorry, I got confused with RDS Phase 2 logic. The rule model will create the triggers

Reply:
Yeah I decided to keep the changes for fixing the create/POST endpoint separate. They'll be merged in by #2736